two_encoder_module/two_encoder_src/two_encoder_src.ino

```diff
@@ -435,7 +440,12 @@ void setup()
     while (!Serial)
     ;
 #endif
-
+  
+    // this is necessary for reconnecting a device because it need some timme for the serial port to get open, but meanwhile the handshake message was send from eos
+    SLIPSerial.beginPacket();
+    SLIPSerial.write((const uint8_t*)HANDSHAKE_REPLY.c_str(), (size_t)HANDSHAKE_REPLY.length());
+    SLIPSerial.endPacket();
+ 
```

Comment:
I tested it with an Arduino Mega with Win10 on Bootcamp. The problem is following;
if you start the EOSnomad application before plug in the USB of Arduino, EOSnomad makes a handshake replay before you can receive messages on the serial port of the Arduino. So, Arduino doesn't know that the handshake replay is send, all the messages from Arduino get know rejected by nomad. With the addition of the few lines (meanwhile I made my own initEOS() function, which add also the filters and subscriptions) you can plug in and out the Arduino without a problem.